run.bash

```diff
@@ -53,14 +67,16 @@ fi
 
 # Run Lava.
 
-config=${LAVA_CONFIG:-"${GITHUB_ACTION_PATH}/default.yaml"}
 output=$(mktemp)
 
-lava scan -c "${config}" > "${output}"
+lava scan -c "${config}" >> "${output}"
 status=$?
 
-echo "status=${status}" >> "${GITHUB_OUTPUT}"
-echo "report=${output}" >> "${GITHUB_OUTPUT}"
+{
+echo "status=${status}"
+echo "report=${output}"
+echo "config=${config}"
+} >> "${GITHUB_OUTPUT}"
```

Comment:
If you are grouping together all the writes, I'd use `>` to make explicit that this is the first (and only) write to `${GITHUB_OUTPUT}`.

```suggestion
{
echo "status=${status}"
echo "report=${output}"
echo "config=${config}"
} > "${GITHUB_OUTPUT}"
```